tools/manifest/custom-elements-manifest.config.js

```diff
@@ -78,6 +86,14 @@ export function createManifestConfig(library = '') {
                 delete declaration.customElement;
               }
 
+              /**
+               * When a generic T type is used in a superclass declaration, it overrides the type defined in derived class
+               * during the doc generation (as the `value` property in the `SbbFormAssociatedMixinType`).
+               * Using the `@overrideType` annotation in the jsDoc's derived class allows to override the type with the correct one.
+               *
+               * In this phase, the script looks for all the `classDeclaration` which have the `overrideTypeKey` property,
+               * and it cycles through them updating the provided property with the provided type.
+               */
```

Comment:
```suggestion
              // Appy the @overrideType
```
Since you extensively explained it in the precedent section, I believe that just a simple connective thought is enough here